atomic_reactor/schemas/config.json

```diff
@@ -652,11 +652,19 @@
             "allOf": [{"$ref": "#/definitions/media_types"}]
           },
           "blacklist_srpms": {
-              "description": "Don't include specified srpms to source container image, specify SRPM name",
-              "type": "array",
-              "items": {
-                "type": "string"
-              }
+              "description": "Don't include srpms specified in blacklist json with key",
```

Comment:
We could keep the former description here to make sure we do say that  the SRPM names are required (and not the RPM names, for example). The json/key behavior is well documented below.

Reply:
added it to that other description